RealmSwift/List.swift

```diff
@@ -432,6 +432,10 @@ public final class List<Element: RealmCollectionValue>: ListBase {
         return List(objc: _rlmArray.freeze())
     }
 
+    public func thaw() -> List? {
```

Comment:
Missing doc comment

Reply:
The doc comment for List is pulled from [AnyRealmCollection](https://github.com/realm/realm-cocoa/pull/6975/files#diff-53ffa4c09fd61e70eef1403c51e39baa5ac2145873029bd5c9916f302a970852R1099). This is symmetric with freeze.
<img width="519" alt="Screen Shot 2021-01-26 at 12 12 18 PM" src="https://user-images.githubusercontent.com/25092858/105879364-12e89200-5fd0-11eb-8bea-0f40371121d8.png">

In an editor, appears like so:
<img width="752" alt="Screen Shot 2021-01-26 at 12 11 43 PM" src="https://user-images.githubusercontent.com/25092858/105879450-314e8d80-5fd0-11eb-92b7-7beaff928fd0.png">
<img width="517" alt="Screen Shot 2021-01-26 at 12 11 50 PM" src="https://user-images.githubusercontent.com/25092858/105879457-31e72400-5fd0-11eb-839b-641a408f3d45.png">